index.html

```diff
@@ -0,0 +1,791 @@
+                    <h4>
+                        <span> by </span> <a href="">
+                            <span> Lorem Ipsum </span>
+                        </a>
+                    </h4>
+                    
+                        <span> Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatum magnam esse, est odit
+                            ullam nemo laboriosam aliquid amet unde, voluptatem tempore! Cum, ut! Minima animi itaque
+                            architecto, corrupti at esse. </span>
+                    </article>
+                    <div class="arrow-button">
+                        <span><i class="fas fa-arrow-right"></i> </span>
+                    </div>
+
+                </div>
+
+                <div class="flex-box-5 classhover">
+                    <h2>
+                        <span>
+                            Lorem Ipsum
+                        </span>
+                    </h2>
+                    <article>
+                    <h2>
+                        <a href="#">
+                            <span> Lorem Ipsum Lorem Ipsum Lorem Ipsum Lorem Ipsum </span>
+                        </a>
+                    </h2>
+                    <h4>
+                        <span> by </span> <a href="">
+                            <span> Lorem Ipsum </span>
+                        </a>
+                    </h4>
+                    
+                        <span> Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatum magnam esse, est odit
+                            ullam nemo laboriosam aliquid amet unde, voluptatem tempore! Cum, ut! Minima animi itaque
+                            architecto, corrupti at esse. </span>
+                    </article>
+                    <div class="arrow-button">
+                        <span><i class="fas fa-arrow-right"></i> </span>
+                    </div>
+
+                </div>
+                <div class="flex-box-4 class2">
+                    <img class="imgmain" src="img/grey-user.png" alt="picture of user">
+                    <h2>
+                        <span> lorem lorem lorem lorem </span>
+                    </h2>
+                    <p>
+                        <span>
+                            Lorem ipsum dolor sit amet consectetur adipisicing elit. Velit non officia ipsa, accusantium
+                            illo nisi quod sequi debitis culpa commodi est soluta dolores ipsam sapiente voluptatibus
+                            expedita, architecto ipsum nostrum.
+                        </span>
+                    </p>
+                    <button type="button">
+                        <span>
+                            Lorem Lorem
+                        </span>
+                    </button>
+                    <p>
+                        <span>
+                            Lorem lorem lorem lorem
+                        </span>
+
+                    </p>
+                </div>
+            </div>
```

Comment:
## Article tag

It uses [`article` tag](https://www.w3schools.com/tags/tag_article.asp) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/design_teardown/article_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540025/))

- Change all elements inside latest post and latest post-2  sections tags to `<article>` tag instead just divs

Reply:
It's done.